contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -118,13 +118,6 @@ script: |
   # Extract the git archive into a dir for each host and build
   for i in ${HOSTS}; do
     export PATH=${BASEPREFIX}/${i}/native/bin:${ORIGPATH}
-    if [ "${i}" = "riscv64-linux-gnu" ]; then
-      # Workaround for https://bugs.launchpad.net/ubuntu/+source/gcc-8-cross-ports/+bug/1853740
-      # TODO: remove this when no longer needed
-      HOST_LDFLAGS="${HOST_LDFLAGS_BASE} -Wl,-z,noexecstack"
-    else
-      HOST_LDFLAGS="${HOST_LDFLAGS_BASE}"
-    fi
```

Comment:
So the reason that this is a workaround is that it's not 100% safe. Normally the object files are supposed to flag that they support non-executable stack. Due to a bug in that verson of RISC-V gcc, this is not signaled.

I'm not sure it's a good idea to set this by default. If it's required for PPC here, we should add a specific exception.